Assignment_3.md

```diff
@@ -0,0 +1,243 @@
+package Core_Java_Assignments;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.ListIterator;
+import java.util.*;
+
+class Employee implements Serializable {
+
+    String Name;
+    String Email_Address;
```

Comment:
variable name should be lower camel case.

Reply:
yes sir